qtpy/tests/test_qtcore.py

```diff
@@ -2,7 +2,7 @@
 
 import pytest
 
-from qtpy import PYQT5, PYQT6, PYSIDE2, PYQT_VERSION, QtCore
+from qtpy import PYQT5, PYQT6, PYSIDE2, PYQT_VERSION, PYSIDE6, QtCore
```

Comment:
```suggestion
from qtpy import PYQT5, PYQT6, PYSIDE2, PYSIDE6, PYQT_VERSION, QtCore
```

Worth being consistant